mecanum_kinematics:
  ros__parameters:
    # URDF dosyasindan hesaplanan gercek degerler
    wheel_radius: 0.1625
    half_length: 0.40
    half_width: 0.395

    # URDF dosyasindan alinan joint isimleri
    front_left_joint: "wheel_fl_joint"
    front_right_joint: "wheel_fr_joint"
    rear_left_joint: "wheel_rl_joint"
    rear_right_joint: "wheel_rr_joint"